### ISSUE TITLE
New DTI Dataset || Cache Challenge

### ISSUE THREAD

**Issue by vivek-booshan** [mims-harvard/TDC#382]
Hey,

There's a DTI challenge website called [cache-challenge](https://cache-challenge.org/critical-assessment-computational-hit-finding-experiments) and at least for challenges 1-4, and was wondering if it would be of interest to integrate them? These are fairly difficult challenges for prediction models and have the added bonus of not having overlapping data with common training sets like bindingdb or pdbbind.

They do provide only a relatively small amount of data, but this dataset would act more as a test/validation set to challenge a prediction model.
